src/components/AddProductForm.js

```diff
@@ -50,14 +50,14 @@ function AddProductForm({onAdd}) {
       </CenterBox>
       <CenterBox>
         <ImageSelector
-          onChange={selected => changeProduct({img: selected.src})}/>
```

Comment:
Passing arrow functions directly as props is not critical, but generally discouraged